src/components/Layout/Layout.js

```diff
@@ -225,10 +225,14 @@ class Layout extends Component {
                       </Menu.Item>
                     </Ref>
                   }
-                  <Menu.Item position="right" className="mobile-hidden">
-                    <DonateNow language={language} />
-                    <VirtualHomeButton language={language} />
-                  </Menu.Item>
+                  {
+                    !isMobileDevice && (
```

Comment:
No donation button in mobile?

Reply:
yes, before we was hide it with _mobile-hidden_ css